### PR TITLE
[GitLab Security Release] Upgrade to 13.9.4

### PR DESCRIPTION
**Pull request by sachilles**
GitLab Patch Release: 13.9.4 (see https://about.gitlab.com/releases/2021/03/16/security-release-gitlab-13-9-4-released/)

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 This file only reflects the changes that are made in this image. Please refer to the upstream GitLab [CHANGELOG](
 https://gitlab.com/gitlab-org/gitlab-foss/blob/master/CHANGELOG.md) for the list of changes in GitLab.
 
+**13.9.3**
```
Comment (Maescool) [inline Changelog.md:6]: typo?

**Comment by sachilles** [comment]
Yes, thanks for the hint. I'll fix it.

**Review by sameersbn — APPROVED** [review]
lgtm

**Comment by sameersbn** [comment]
waiting for the CI to report the results. PR will be merged soon..

**Pull request merged**